### PR TITLE
cloud example

### PR DESCRIPTION
**Pull request by boucadair**

### CODE REVIEW

File: draft-ietf-opsawg-teas-attachment-circuit.md [hunk 1]
```diff
@@ -1250,23 +1250,23 @@ The attachment circuit in this case use a SAP identifier to refer to the physica
...
 
-Firstly, Figure {{cloud-parent-infra}} describes the physical infrastructure on which attachment circuits are provisionned. The compute nodes (customer) are attached to the provider infrastructure thanks to a set of bearers (i.e. compute-XX-nicY). The provider infrastructure can be realized in multiple ways, such as  IP Fabric, Gateways or L2/L3 Edge Routers. This document does not intend to detail these aspects.
+{{cloud-parent-infra}} describes the physical infrastructure on which attachment circuits are provisionned. The compute nodes (customer) are attached to the provider infrastructure via a set of bearers (i.e., "compute-XX-nicY"). The provider infrastructure can be realized in multiple ways, such as IP Fabric, Layer 2/Layer 3 Edge Routers. This document does not intend to detail these aspects.
```
Comment (boucadair) [inline draft-ietf-opsawg-teas-attachment-circuit.md:1264]:
```suggestion
{{cloud-parent-infra}} describes the physical infrastructure on which attachment circuits are provisioned. The compute nodes (customer) are attached to the provider infrastructure via a set of bearers (i.e., "compute-XX-nicY"). The provider infrastructure can be realized in multiple ways, such as IP Fabric, Layer 2/Layer 3 Edge Routers. This document does not intend to detail these aspects.
```

**Comment by robric** [comment]
Guys, this is a draft, please wait before commenting.



Juniper Business Use Only

**Comment by boucadair** [comment]
> Guys, this is a draft, please wait before commenting

That was not clear because the text commented by Samier was already in the main copy. Please keep the draft version in a separate PR for future changes. Thanks.

**Comment by robric** [comment]
yeah I know, I am struggling to handle branching from the vscode  github plugin ☹
Sorry for that !
R



Juniper Business Use Only

**Pull request closed without merging**